code/go/0chain.net/blobber/flags.go

```diff
@@ -47,10 +47,6 @@ func parseFlags() {
 	fmt.Print("> load flags")
 	flag.Parse()
 
```

Comment:
It is checked in config.go because mount point can be given from config.yaml file as well.
Should we remove it from config.yaml file, then we need to check it here